raiden/ui/cli.py

```diff
@@ -205,16 +211,15 @@ def app(address,
 
         address = addresses[idx]
 
-    password = os.environ.get('ETH_PASSWORD')
-    # if environ has password, try it
-    # avoid using click method to avoid password in command line
+    password = None
+    if password_file:
+        password = password_file.read().splitlines()[0]
```

Comment:
I wonder when does `click` close the file. According to [this](http://click.pocoo.org/5/arguments/#file-opening-safety) it does so in an "intelligent way" but that does not tell me much. Do you have any idea when `password_file` would be closed?